tests/pages/encryption-password-popup.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class EncryptionPasswordPopup {
```

Comment:
EncryptionSettingsPopup (in the future it could be something else besides password, why to change the name...).

Reply:
That was what I initally had, but then I thought... hey its just the password. :-)